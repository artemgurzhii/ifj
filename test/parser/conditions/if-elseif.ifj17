if a then
  return 1
elseif a then
  return 2
elseif a then
  return 3
end
