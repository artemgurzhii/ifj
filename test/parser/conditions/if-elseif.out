(if (> (int 1) (int 0))
  (return
    (int 1)))
(else if (> (int 2) (int 1))
  (return
    (int 2)))
(else if (> (int 3) (int 2))
  (return
    (int 3)))
